src/features/courses/course-route.ts

```diff
@@ -214,6 +202,18 @@ router.get('/questions',
             userId = userIdInput;
         }
 
+        let topic: CourseTopicContent | null = null;
+        if(!_.isNil(req.query.courseTopicContentId)) {
+            topic = await courseController.getTopicById(req.query.courseTopicContentId, userId);
+            if (new Date().getTime() < topic.startDate.getTime()) {
```

Comment:
Nit: We should keep this consistent, if we use moment in some places we should stick with moment

Reply:
This was old code, but I agree. I'll patch it up.